### ISSUE TITLE
Resources

### ISSUE THREAD

**Issue by Philnett1970** [Philnett1970/WinTility#1]
Guss first thing is to start gathering all resources, that will be needed for the project. what do you think? @seankelley1970

**Comment by seankelly1970** [comment]
Yeah I think that that would he a good idea. I need to get my head in the right place and start learning this new coding so that I can help out. Going to he busy for the next few days though, however am going to try and do some brain storming and come up with some fresh ideas. Have feeling this coding is different from DW so would be no point looking for that coding. But definitely thi k that this project should be based loosely on our original project but with a few tweaks here and there. I think that we should also include a speed test of some kind so that people can check if they are getting the internet speeds that they are supposed to. I also think that a nice ping feature should be included, one that once the command is executed the window would auto close so that the inexperienced can't do anything from the command prompt. What do you think @phillnett1970 ?
